### PR TITLE
Update to include  V70 support

### PR DESCRIPTION
**Pull request by sgbhargava**

### CODE REVIEW

**Comment by sgbhargava** [comment]
@ryanvergel Please let me know if there has been any updates to the themes or templates

**Comment by sgbhargava** [comment]
Thanks @ryanvergel I have updated the theme

File: master/management-specification/doc/appendix_a.rst [hunk 1]
```diff
@@ -61,6 +62,8 @@ The table below captures the PCIe information for the following Alveo™ product
 +-------------------------+----------------------------------------------+
 | VCK5000 QDMA            | PF0=0x5048 PF1=0x5049 PF2=0x504A PF3=0x504B  |
 +-------------------------+----------------------------------------------+
+| V70 QDMA                | PF0=0x50B0 PF1=0x50B1 PF2=0x50B2 PF3=0x50B3  |
++-------------------------+----------------------------------------------+
```
Comment (krishna-xilinx) [inline master/management-specification/doc/appendix_a.rst:66]: Edit line 65 as V70PQ2 QDMA
Add another line/row before V70PQ2. 
```
+-------------------------+----------------------------------------------+
V70 QDMA                | PF0=0x5094 PF1=0x5095 PF2=0x5096 PF3=0x5097
+-------------------------+----------------------------------------------+
```

**Review by krishna-xilinx — CHANGES_REQUESTED** [review]
pls add V70PQ2 PCIe info as mentioned

**Review by krishna-xilinx — APPROVED** [review]

**Review by krishna-xilinx — APPROVED** [review]

**Pull request merged**